index.mjs

```diff
@@ -0,0 +1,3 @@
+import mod from './index.js';
+
+export const StyleToObject = mod;
```

Comment:
Should we be consistent with how the package does CommonJS export here?

```js
export default mod;
```

Reply:
ok no prob, will replace with default export